src/pages/index.tsx

```diff
@@ -46,9 +48,17 @@ const Home: NextPage = () => {
           Philippines. 📚💜💻{" "}
         </p>
 
+        {!!directories && (
+          <p className="mb-[20px] text-center text-base leading-6 text-neutral-light md:mb-[50px] md:text-left">
```

Comment:
can we try a lighter shade for this? need to emphasize the description of the site more than this text